### ISSUE TITLE
[DOCS] Improve Changelog entry for static asset routes

### ISSUE THREAD

**Issue by TYPO3IncTeam** [TYPO3-Documentation/Changelog-To-Doc#988]
:information_source: View this commit [on Github](https://github.com/TYPO3/typo3/commit/6c1c51a961ad43c3d9ce226dc0dae43cca86a32d)
:busts_in_silhouette: Authored by Garvin Hicking gh@faktor-e.de
:heavy_check_mark: Merged by Stefan Bürk stefan@buerk.tech

## Commit message

[DOCS] Improve Changelog entry for static asset routes

Highlight site-specific routing, fix spelling error,
add code snippet caption.

Resolves: #104580
Releases: main
Change-Id: I46527bf93fbf879e1a45e254e72e26f66c397eaf
Reviewed-on: https://review.typo3.org/c/Packages/TYPO3.CMS/+/85570
Reviewed-by: Benni Mack <benni@typo3.org>
Reviewed-by: Stefan Bürk <stefan@buerk.tech>
Tested-by: Benni Mack <benni@typo3.org>
Tested-by: core-ci <typo3@b13.com>
Tested-by: Stefan Bürk <stefan@buerk.tech>

## :heavy_division_sign: Modified files

<details>
<summary>13.3/Feature-101472-AllowStaticRoutesToAssets.rst</summary>

```diff
@@ -15,17 +15,22 @@ It is now possible to configure static routes with the type `asset` to link to
 resources which are typically located in the directory
 :file:`EXT:my_extension/Resources/Public/`.
 
-.. code-block:: yaml
+..  code-block:: yaml
+    :caption: config/sites/my-site/config.yaml
 
     routes:
       -
         route: example.svg
         type: asset
         asset: 'EXT:backend/Resources/Public/Icons/Extension.svg'
 
+Note that the asset URL can be configured on a per-site basis.
+This allows to deliver site-dependent custom favicon or manifest
+assets, for example.
+
 Impact
 ======
 
-Static routes to files shipped with extensions can now be configued in the site configuration.
+Static routes to files shipped with extensions can now be configured in the site configuration.
 
 .. index:: Frontend, YAML, ext:frontend
```

</details>

**Comment by garvinhicking** [comment]
Reopening - we should add/document this for:

https://docs.typo3.org/m/typo3/guide-installation/12.4/en-us/MigrateToComposer/AssetMigration.html

as well

**Comment by linawolf** [comment]
I dont think so, the delivery of assets via routes does not depend on the installation method. There is no reference to that in the changelog entry or the referenced change
